packages/combobox/src/Combobox.types.ts

```diff
@@ -305,6 +306,16 @@ interface BaseComboboxOptionProps extends ComponentPropsWithoutRef<'li'> {
    * Styling Prop
    */
   className?: string;
+
+  /**
+   * Optional descriptive text under the displayName
+   */
+  description?: string;
+
+  /**
+   * Callback fired when an option is clicked. Returns the option value.
+   */
+  onClick?: (value: string) => void;
```

Comment:
Should this pass a `ClickEvent` instead of the value?

Reply:
yes it should - let's be consistent about event handler signatures